docs/.vuepress/enhanceApp.js

```diff
@@ -26,6 +26,7 @@ import KToggle from '../../packages/KToggle/KToggle.js'
 import Komponent from '../../packages/Komponent/Komponent.js'
 import KSlideout from '../../packages/KSlideout/KSlideout.vue'
 import KSkeleton from '../../packages/KSkeleton/KSkeleton.vue'
+import KSkeletonBox from '../../packages/KSkeleton/KSkeletonBox.vue'
```

Comment:
I’m wondering if this should just be a helper class instead. I wonder if adding another component just to add classes is worth the trouble? Like really we’re just adding the ability to add whatever width and height div inside the card right?

Reply:
Though at the end of the day I guess it doesn’t matter. We’re just exposing the underlying div. 🤔

Reply:
the idea is that we can describe our own DSL outside the context of css and it will allow users to easily create placeholder skeletons themselves without having to think too much about how it's implemented under the hood.